server/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -51,17 +51,18 @@ public Content render(Params params) {
             .with(div(renderBlockWithSubmitForm(params)).withClasses(Styles.MY_8))
             .withClasses(Styles.MY_8, Styles.M_AUTO);
 
+    String pageTitle =
+        params.programTitle()
+            + " — "
+            + (params.blockIndex() + 1)
+            + " of "
+            + params.totalBlockCount();
+
     HtmlBundle bundle =
         layout
             .getBundle()
-            .setTitle(params.programTitle())
-            .addMainContent(
-                h1(params.programTitle()
-                        + " "
-                        + (params.blockIndex() + 1)
-                        + " of "
-                        + params.totalBlockCount())
-                    .withClasses(Styles.SR_ONLY))
+            .setTitle(pageTitle)
```

Comment:
This is used here: https://github.com/civiform/civiform/blob/main/server/app/views/applicant/ApplicantLayout.java#L89